artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2188,8 +2189,16 @@
       <xsd:attribute name="source-mirror-address" type="xsd:string" use="optional" default="">
          <xsd:annotation>
             <xsd:documentation>
-               By default the replica will use a temporary store and forward queue to store events towards the mirror / replica.
-               However if this is set, we will use a defined durable queue.
+               Deprecated: this is ignored from any parsing.
+               The xsd is allowing this for compatibility reasons. This will be eventually removed from the codebase.
```

Comment:
If its ignored, then wont it just leave them with a a queue, plus broken not-mirroring of any pending stuff? Seems better to fail startup. I would remove it then.

Reply:
Ok, I actually added because of your concern on the XSD. there's not much I can do there. I will remove it then.

Reply:
I thought it seemed unecessary to remove it, as it wasnt clear how a user would handle the resulting failure.

However adding it and then ignoring it is definitely worse still, as it makes people think its still being used when it isnt, perhaps skipping mirroing some stuff. Better to fail at that point so they know there is a change and potentially problems with it.